lib/apply.js

```diff
@@ -2,6 +2,49 @@
 
 import rest from 'lodash/rest';
 
+/**
+ * Creates a continuation function with some arguments already applied.
+ *
+ * Useful as a shorthand when combined with other control flow functions. Any
+ * arguments passed to the returned function are added to the arguments
+ * originally passed to apply.
+ *
+ * @name apply
+ * @static
+ * @memberOf async
+ * @category Util
+ * @param {Function} function - The function you want to eventually apply all
+ * arguments to. Invokes with (arguments...).
+ * @param {...*} arguments... - Any number of arguments to automatically apply
```

Comment:
Is this the way it should be defining this? /cc @jdalton

This appears to be the only thing being parsed incorrectly at the moment using jscs-jsdoc (see the linked commit)

http://stackoverflow.com/questions/4729516/correct-way-to-document-open-ended-argument-functions-in-jsdoc